frontend/src/packages/trucks/libs/validation-schemas/truck-add.validation-schema.ts

```diff
@@ -0,0 +1,73 @@
+import joi from 'joi';
+
+import {
+  FormLabel,
+  TruckCapacity,
+  TruckManufacturer,
+  TruckPricePerKm,
+  TruckTowType,
+  TruckYear,
+} from '../enums/enums.js';
+import { type TruckFormModel } from '../types/types.js';
+
+const truckAddValidationSchema = joi.object<TruckFormModel, true>({
```

Comment:
do you need this validation schema only on fe? 
seems it's almost repeating truckCreateRequestBodyValidationSchema, except drivers field. I think we need avoid duplicating such large block of code

Reply:
only for fe form